app/controllers/users_controller.rb

```diff
@@ -50,6 +57,18 @@ def destroy
     redirect_to users_path
   end
 
+
+  def import
+    result = User.import(params[:attachment])
+    unless result == false
```

Comment:
why don't you use if instead of unless ?

Reply:
done